Packs/Reco/Integrations/Reco/Reco.yml

```diff
@@ -0,0 +1,49 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Reco
+  version: -1
```

Comment:
I think that the version is 1.0.1?

In any case worth setting up https://pypi.org/project/bump2version/ (the version string appears in multiple places) or even a `bump_version.sh` script

Reply:
version: -1 
default